projects/solar/src/relay_fsm.c

```diff
@@ -2,17 +2,64 @@
 
 #include <stdbool.h>
 
+#include "can_transmit.h"
+#include "can_unpack.h"
+#include "data_store.h"
 #include "event_queue.h"
 #include "fault_handler.h"
 #include "fsm.h"
+#include "gpio.h"
 #include "log.h"
+#include "solar_boards.h"
 #include "solar_config.h"
 #include "solar_events.h"
 #include "status.h"
 
+static SolarMpptCount solar_mppt_count;
+
+static void prv_assert_relay(SoftTimerId timer_id, void *context) {
+  RelayFsmStorage *storage = (RelayFsmStorage *)context;
+  if (storage->isErrCalled == true) {
```

Comment:
[Boolean zen](https://courses.cs.washington.edu/courses/cse143/20sp/style/zen): you don't need the `== true`.

Reply:
done